webapp/src/main/java/ar/edu/itba/paw/webapp/controllers/UserController.java

```diff
@@ -1,185 +1,55 @@
-			return new ModelAndView(UNAUTHORIZED);
-		}
-
-		if (errors.hasErrors()){
-			redirectAttributes.addFlashAttribute("org.springframework.validation.BindingResult.resetPasswordForm", errors);
-			redirectAttributes.addFlashAttribute("resetPasswordForm", passwordForm);
-
-			final String referrer = request.getHeader("referer");
-			return new ModelAndView("redirect:" + referrer);
-		}
-
-		final Authentication auth = SecurityContextHolder.getContext().getAuthentication();
-		if (auth == null) {
-			LOGGER.warn("User {} tried to change a password and auth was null [POST]", loggedUser.getDni());
-			return new ModelAndView("redirect:/errors/401");
-		}
-		UserSessionDetails user = (UserSessionDetails) auth.getPrincipal();
-
-		/* Check that the DNI is still valid and was not modified */
-		if (!String.valueOf(passwordForm.getDni()).equals(user.getUsername()) && !user.hasAuthority("ADMIN")) {
-			LOGGER.warn("Someone tried to reset a password from another user.");
-			LOGGER.warn("Logged user's dni: {}", user.getUsername());
-			LOGGER.warn("Victim's dni: {}", passwordForm.getDni());
-
-			return new ModelAndView("redirect:/errors/401");
-		}
-
-		final boolean done = userService.resetPassword(passwordForm.getDni());
-
-		if (!done) {
-			LOGGER.warn("User {} could not reset password, Result = {}", loggedUser.getDni(), done);
-			redirectAttributes.addFlashAttribute("alert", "danger");
-			redirectAttributes.addFlashAttribute("message", messageSource.getMessage(
-					"INVALID_PASSWORD", null, Locale.getDefault()));
-		} else {
-			LOGGER.info("User {} reset password successfully", loggedUser.getDni());
-			redirectAttributes.addFlashAttribute("alert", "success");
-			redirectAttributes.addFlashAttribute("message", messageSource.getMessage("change_pwd_success",
-					null,
-					Locale.getDefault()));
-		}
-
-		final String referrer = request.getHeader("referer");
-		return new ModelAndView("redirect:" + referrer);
-	}
-
-	@RequestMapping(value = "/user/changePassword", method = RequestMethod.POST)
-	public ModelAndView changePassword(
-			@Valid @ModelAttribute("changePasswordForm") final PasswordForm passwordForm,
-			final BindingResult errors,
-			final RedirectAttributes redirectAttributes) {
-		LOGGER.info("User {} is about to change password", passwordForm.getDni());
-		passwordValidator.validate(passwordForm, errors);
-
-		if (errors.hasErrors()){
-			LOGGER.warn("User {} could not change password due to {} [POST]", passwordForm.getDni(), errors.getAllErrors());
+@Component
+@Path("/users")
+public class UserController {
 
-			/* set alert */
-			redirectAttributes.addFlashAttribute("alert", "danger");
-			redirectAttributes.addFlashAttribute("message", messageSource.getMessage("formWithErrors", null, Locale.getDefault()));
-			/* --------- */
+  private final static Logger LOGGER = LoggerFactory.getLogger(UserController.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)